resources/views/livewire/price-ticker.blade.php

```diff
@@ -1,10 +1,10 @@
 @php ($isDisabled = ! Network::canBeExchanged() || config('explorer.network') !== 'production' || ! $isAvailable)
 
 <div
-    wire:poll.60s
+    wire:poll.visible.60s
```

Comment:
isn't it always visible in the navbar? 🤔 or does this aim at mobile

Reply:
there are 2 instances of the component, 1 for desktop and 1 for mobile. Without this change, both components poll every 60 seconds (2 requests). With this change, only the visible component is polled - either the desktop navbar or the mobile navbar (if it has been opened). Thinking about it, is it worth dropping the poll frequency for the mobile navbar dropdown since it's not open very long generally :thinking:

Reply:
can leave it at 60 if it's only polling when visible anyway